core-features/studio/test.ts

```diff
@@ -1,13 +1,18 @@
 import fs from 'fs'
 import { createHash } from 'crypto'
 import fetch from 'node-fetch'
+import { PrismaClient } from '@prisma/client'
 
 const STUDIO_PORT = 5555
 const SCHEMA_HASH = createHash('md5')
   .update(fs.readFileSync('./prisma/schema.prisma'))
   .digest('hex')
 
+const prisma = new PrismaClient()
```

Comment:
Having that here has the danger, that the binaries are downloaded for the normal PrismaClient usage instead of for Studio, right?

Reply:
Yes, but the assertions in test.sh ensures nothing extra is downloaded.

Regardless, I got rid of it because it felt wrong. I'm now ousing Studio to `findMany` after a `create`.